src/style.css

```diff
@@ -1 +1,83 @@
 /* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav_menu {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+
+  list-style: none;
+  margin: 0 10px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  padding: 23px 0;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.nav_link:hover::after {
+  text-decoration: underline;
+}
```

Comment:
Do you really need this selector? It's redundant I think. It doesn't do anything.